guides/tasks/INC_ASG_BOXES.md

```diff
@@ -0,0 +1,47 @@
+## Adjusting the number of ASG boxes
```

Comment:
```suggestion
## Adjusting the size of an ASG
```

Reply:
I don't like the filename - `INC` is an unnecessary abbreviation,
how about `RESIZING_AN_ASG.md` ?

Reply:
adjusted as per discussion elsewhere